website/docs/docs/dbt-cloud-apis/user-tokens.md

```diff
@@ -33,18 +33,18 @@ The migration to PATs is critical if you are using user API keys today. The curr
  There are a few things to understand if you are using a user API key today: 
 
 * Personal access tokens are more secure. 
-    * To promote least privilege and high security assurance for your dbt Cloud accounts, we highly recommend moving to the new account-scoped personal access tokens.
+    * To promote the least privilege and high-security assurance for your dbt Cloud accounts, we highly recommend moving to the new account-scoped personal access tokens.
 * You must create and use unique tokens in each one of your dbt Cloud accounts that share the same email address.
-    * For example, if paul@atreides.com belongs to two dbt Cloud accounts: Spice Harvesting Account and Guild Navigator Account. Prior to this release, the same API key was used to access both of these accounts. 
+    * For example, if paul.atreides@example.com belongs to two dbt Cloud accounts: Spice Harvesting Account and Guild Navigator Account. Before this release, the same API key was used to access both of these accounts. 
     * After this release, Paul has to individually go into these accounts and create a unique PAT for each account he wants to access the API for. These PATs are account-specific and not user specific. 
 * Cross-Account API endpoints will change in behavior when using the personal access tokens.
-    * These are namely /v2/accounts and /v3/accounts. Since all PATs are now account specific, getting all accounts associated to a username cannot work. /v3/accounts will only return account metadata that’s relevant to the PAT that’s being used. 
+    * These are namely /v2/accounts and /v3/accounts. Since all PATs are now account specific, getting all accounts associated with a username cannot work. /v3/accounts will only return account metadata that’s relevant to the PAT that’s being used. 
     * User account metadata will only contain information about the specific account under which the request is being made. 
     * Any other accounts that belong to that user account will need to be requested through the PAT that belongs to that account. 
 
 :::warning Undocumented APIs
 
-If you’re using any undocumented and unsupported API endpoints, please note that these can be deprecated without any notice. If you are using any undocumented endpoints and have use-cases that are not satisfied by the current API, please reach out to [support@dbt.com](mailto:support@dbt.com). 
+If you’re using any undocumented and unsupported API endpoints, please note that these can be deprecated without any notice. If you are using any undocumented endpoints and have use-cases that are not satisfied by the current API, please reach out to [support@getdbt.com](mailto:support@getdbt.com). 
```

Comment:
hey Matt, support@getdbt.com is the preferred support email. i believe support@dbt.com doesn't work (tested it with support) and they have raised this with cs leadership.